### ISSUE TITLE
Improve the top of the newsletter

### ISSUE THREAD

**Issue by chriscool** [git/git.github.io#59]
I'd rather have just something shorter at the top of each edition that redirect to http://git.github.io/rev_news/rev_news.html more strongly for all the meta information. And in turn we could have more and better meta information, like a nicer signup, on http://git.github.io/rev_news/rev_news.html.

For example if I would rewrite edition 2, I would only write something like (without showing the links) at the top:

```
Git Rev News: Edition 2 (April 15, 2015), 10 years of Git & Git Merge 2015!

Welcome to the second edition of Git Rev News, a digest of all things Git.
For our goals, the archives, the way we work, how to contribute or to
subscribe see the Git Rev News page on http://git.github.io.

This special edition covers Git's 10th year of existence, as well as the Git
Merge conference held on April 8th & 9th in Paris, France. Git developers
and users alike came together to celebrate the anniversary, and to discuss
the current challenges of using and scaling Git.

Discussions

General

10 years of fun

...

```

**Comment by tfnico** [comment]
Thanks for bringing this up. I'm pretty sure that the more we shove the signup form in the user's face, the more signups we will get. From what I hear and read about website UX these days, it's the winner way to get returning traffic. Most people don't subscribe to RSS anymore, and most people will spend less than a few seconds browsing the page before moving on. A very visible sign-up form could help a lot.

Of course, we should have a nice sign-up form on the about page too. We can start with this, and then play around with the templates to see what it would look like in the header of an edition.

**Comment by chriscool** [comment]
@tfnico I agree that having the signup form in the user face might be a good thing to get a lot of traffic fast. But I think in the long run it is better to make the top as less anoying as possible for the reader and that means that we should put more content and less meta content at the top. 

For example if you look at LWN.net, except the menu, you will see only the following about LWN itself:

```
What is LWN.net?
LWN.net is a reader-supported news site dedicated to producing the best coverage from within the Linux and free software development communities. See the LWN FAQ for more information, and please consider subscribing to gain full access and support our activities. 
```

If you look at for example http://www.washingtonpost.com/, http://time.com/, http://www.nytimes.com/ or http://www.lemonde.fr/ you will not see much meta content, except on the nytimes that has a lot of subscribe advertising on it but no other advertising.

As I think a signup form is a bit anoying, my opinion is that it is better to do as LWN does and have a few lines with some links, one or more of these links being subscribe links.

**Comment by tfnico** [comment]
@chriscool OK, how about a signup form at the bottom of each editition?

**Comment by chriscool** [comment]
@tfnico ok with a signup at the bottom. Can you take care of it?
I will rework the top as I suggested above, but I am still open to suggestions about it.

**Comment by tfnico** [comment]
@chriscool Yup, but I think my top prio is to fix #45 and add some #57 first.

**Comment by chriscool** [comment]
@tfnico ok no problem.
I created issue #60 to add a signup form at the bottom.

I close this as the following commits took care of this issue:

a049cc7fec7a0bb66dd87911d4cf4332549ac61e
6cb3f8c585a72d6bbe7fd42d4833cb0027b17328
22e973926aeaeec12fdf378f78d08bd78ff873d3
a20d18e3a4b05b23098c4409674e8359371902ab
